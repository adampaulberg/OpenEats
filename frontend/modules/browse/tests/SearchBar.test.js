import React from 'react';
import SearchBar from '../components/SearchBar';
import createComponentWithIntl from '../../../jest_mocks/createComponentWithIntl';

test('Loading component test', () => {
  const component = createComponentWithIntl(
    <SearchBar/>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
