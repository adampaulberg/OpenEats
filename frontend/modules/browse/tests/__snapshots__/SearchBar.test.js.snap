// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component test 1`] = `
<div
  className={undefined}
>
  <div
    className="input-group search-bar"
  >
    <span
      className="input-group-addon"
      id="search_bar_label"
    >
      <span
        className="hidden-xs"
      >
        Search for Recipes
        :
      </span>
      <span
        className="visible-xs"
      >
        Search
        :
      </span>
    </span>
    <input
      aria-describedby="search_bar_label"
      className="form-control"
      name="SearchBar"
      onBlur={[Function]}
      onChange={[Function]}
      onKeyDown={[Function]}
      placeholder="Enter a title, tag, or ingredient"
      type="text"
      value=""
    />
    
  </div>
</div>
`;
